compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/JvmGeneratorExtensions.kt

```diff
@@ -65,6 +65,9 @@ object JvmGeneratorExtensions : GeneratorExtensions() {
             descriptor.visibility
         else
             null
+
+    override fun isPropertyWithPlatformField(descriptor: PropertyDescriptor): Boolean =
+        descriptor.hasJvmFieldAnnotation()
```

Comment:
I don't really like the idea of copying this, though. Maybe it's best to merge the two extension classes instead?

Reply:
Yeah, it'd make sense to merge them. I've separated them initially after thinking that they might have different lifecycles, but it doesn't seem to be the case.

Reply:
There were some problems with module boundaries, so instead of moving the entire GeneratorExtensions to org.jetbrains.kotlin.ir.util org.jetbrains.kotlin.ir.util I simply made them inherit from StubGeneratorExtensions.